src/app/shared/page/header/header.component.html

```diff
@@ -0,0 +1,101 @@
+<div class="pb-24 lg:pb-48">
+  <div class="fixed w-full z-50 inset-x-0 top-0 bg-white pb-24">
+    <div class="relative">
+      <div
+        class="absolute inset-x-0 top-0 h-2 bg-gradient-to-r from-teal-500 to-blue-800"
+      >
+        <!-- navbar section -->
+        <nav class="container md:flex md:justify-between md:items-center">
+          <div
+            class="flex items-center justify-between p-4 gap-12 text-lg text-blue-950"
+          >
+            <a
+              [routerLink]="['']"
+              routerLinkActive="font-bold"
+              class="flex items-center md:pl-24 lg:pl-0"
+            >
+              <img src="assets/logo.svg" alt="logo-icon" />
+              <div class="text-teal-400 font-black text-sm pr-2">
+                <P> SHOGOL </P>
+                <P>شـــغـــل</P>
+              </div>
+            </a>
+
+            <!-- Toggle button -->
+            <button class="sm:block md:hidden lg:hidden" (click)="toggleMenu()">
+              <i
+                [ngClass]="
+                  isMenuOpen
+                    ? 'fa-solid fa-xmark fa-2xl transition-all'
+                    : 'fa-solid fa-bars fa-lg transition-all'
+                "
+              ></i>
+            </button>
+            <!-- Navigation links -->
+            <ul
+              class="flex flex-col md:flex-row md:gap-5 md:items-center md:z-auto md:static bg-white items-end w-full absolute left-0 p-6 pt-0 md:p-0 top-20 md:top-0"
+              [class]="isMenuOpen ? 'flex' : 'hidden'"
```

Comment:
use `[ngClass]` instead

```suggestion
              [ngClass]="isMenuOpen ? 'flex' : 'hidden'"
```